src/main/definitions/complexTypes/claimantOtherType.ts

```diff
@@ -0,0 +1,17 @@
+import { PayInterval, WeeksOrMonths, YesOrNo, YesOrNoOrNotSure } from '../case';
+
+export interface ClaimantOtherType {
```

Comment:
I know it translates to this in the backend but let's give this interface a more meaningful name as its difficult to understand which properties should belong to ClaimantOtherType. 
I think we should call it ClaimantEmploymentDetails. If you could update the interface and filename that would be great.

Reply:
I agree, changed it